docs/compatibility.asciidoc

```diff
@@ -28,7 +28,9 @@ that not all core Node.js API's can be instrumented without the use of Async Hoo
 [[elastic-stack-compatibility]]
 === Elastic Stack Compatibility
 
-This agent is compatible with {apm-server-ref}[APM Server] v6.2 and above.
+Version 1.x of the agent is compatible with {apm-server-ref-64}[APM Server] v6.2 to v6.4.
+For support for APM Server v6.5 and above,
+use version 2.0.0 or higher of the Node.js agent.
```

Comment:
@bmorelli25 it might be a good idea to link to the 2.x version of this page. What's the best way to do that? (keep in mind this page have been renamed in 2.x and is now called "supported technologies".

Reply:
One way is to create another shared attribute for `:apm-node-ref-2x`: and link to `{apm-node-ref-2x}/supported-technologies.html`. I think a better way would be to use `{branch}`, but I'm still working through the details of how to implement that with Lisa.

Reply:
New idea:

How about that I create a link like this:

```suggestion
use {apm-node-ref-index}[version 2.0.0 or higher] of the Node.js agent.
```

Where `apm-node-ref-index` is defined as this:

```
:apm-node-ref-index:   https://www.elastic.co/guide/en/apm/agent/nodejs
```

That will take users to the version overview page and they can choose the version they want

Reply:
That's super cool. I like that idea. Are you thinking you'll define `apm-node-ref-index` in the `shared/attributes` file (docs repo), or in the `index.asciidoc` file within the node.js repo?

Reply:
I was thinking to define it in the shared attributes file next to the other `apm-node-ref*` attributes. Is that ok?

Reply:
I was thinking since it most likely won't be used outside of the node documentation it might make more sense to just define it locally. I have a call with Lisa today - I'll discuss this with her and get back to you!

Reply:
Ah ok, let me know what you think is best then after the call 👍

Reply:
`:apm-node-ref-index:` in `shared/attributes` is a 👍 
In addtion, elastic/apm#19 will resolve the other problems we're seeing in this PR.

Reply:
PR created: https://github.com/elastic/docs/pull/479